### PR TITLE
[BYT/CHT] topology: remove SRC, low-latency pipeline and media pipeline

### PR DESCRIPTION
**Pull request by plbossart**
Tentative proposal to remove 'broadwell' topology and instead use same concepts as Merrifield switch matrix or closed-source pipelines starting with a mixer.

Tested on CHT w/ rt5640 and max98090 (no change to kernel and UCM files)

### CODE REVIEW

**Comment by lgirdwood** [comment]
Jenkins flood test fail fail is on one BYT, the other BYT is OK.

**Comment by plbossart** [comment]
> Jenkins flood test fail fail is on one BYT, the other BYT is OK.

well it fails when this PR is used - with a codec. The nocodec mode was not changed.

I am probably missing PIPELINE and scheduling definitions, but they are far from clear to be honest. There are so many ingrained assumptions in those topologies that it's hard to change their structure - a bit of a house of cards if I am honest.

File: tools/topology/sof-byt-codec.m4 [hunk 1]
```diff
@@ -47,39 +55,42 @@ PIPELINE_PCM_ADD(sof/pipe-low-latency-capture.m4,
...
+# 	48000, 48000, 48000,
+# 	SCHEDULE_TIME_DOMAIN_DMA,
+# 	PIPELINE_PLAYBACK_SCHED_COMP_1)
 
 # Connect pipelines together
 SectionGraph."PIPE_NAME" {
 	index "0"
 
 	lines [
 		# media 0
```
Comment (ranj063) [inline tools/topology/sof-byt-codec.m4:80]: @plbossart looks good except for the stale comment above. It is no longer the media pipeline

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
Lets see if the fail is reproducable.

**Comment by plbossart** [comment]
SOFCI TEST

**Comment by plbossart** [comment]
wrong PR, closing

**Pull request closed without merging**